backend/app/controllers/api/v1/project_developers_controller.rb

```diff
@@ -3,6 +3,9 @@ module V1
     class ProjectDevelopersController < BaseController
       include API::Pagination
 
+      skip_before_action :require_json!, only: :create
```

Comment:
I had to upload picture file for project developer so I have decided to go with `multipart/form-data` content type instead.